src/styles/_variables.scss

```diff
@@ -46,5 +46,6 @@ $light-font: 300;
 $default-font-weight: 400;
 
 // font size
+$newuser-text-size: 1.2rem;
```

Comment:
this is very specific, maybe this could be renamed to something a bit more generic? so it could be used elsewhere in the future

Reply:
changed to $larger-text-size